cookbooks/hdfsdu/templates/default/coordinator.properties.erb

```diff
@@ -0,0 +1,32 @@
+jobTracker=<%= node[:hdfsdu][:jobtracker] %>
```

Comment:
Despite the archaic name can this take a modern YARN HA cluster-id? I would prefer not to use host names if we can.